tests/Paramore.Brighter.Core.Tests/TestHelpers/DummyMessageProducer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+
+namespace Paramore.Brighter.Core.Tests.TestHelpers
+{
+    public class DummyMessageProducer : IAmAMessageProducer
```

Comment:
We are trying to get away from using a dummy in a test, in favor of using a Brighter type that is in-memory; in this case there is an InMemoryProducer, which will need an InMemoryProducerFactory. So if we could add that factory into Paramore.Brighter and then create an InMemoryProducerFactory that would complete the circle here.